.circleci/config.yml

```diff
@@ -340,11 +288,14 @@ workflows:
                 - "Finfet"
                 - "Bulk"
               design:
-                - "ota and not OTA_FF_2s_v3e"
-                - "BUFFER and not BUFFER_VCM"
+                # - "ota and not OTA_FF_2s_v3e"
+                # - "BUFFER and not BUFFER_VCM"
+                # - "single_SAR and not tb_single_SAR"
+                # - "Sanitized_Coarse_Comp_CK or Sanitized_EdgeComparator or BTSW or Sanitized_ResTune_Configure and not Sanitized_ResTune_Configure"
+                - "BUFFER or ota"
+                - "San or BTSW"
+                - "not SAR and not San and not BUFFER and not DLL and not ota and not single_to_differential_converter and not COMPARATOR_2LEVEL_BIDIRECTIONAL_MAC_SKEW and not CTD"
                 - "single_SAR and not tb_single_SAR"
```

Comment:
Can you review this list to ensure the commented lines can be completely removed?

These are the testcases being run nightly (includes nightly and full_after_nightly hence the repetition from what you were calling build-integration1 and build-integration2)

Reply:
Yes we can remove them.